### DISCUSSION TITLE
Multiple select dropdown with Livewire and Tailwind?

### DISCUSSION THREAD

**Discussion by toxifiedmatrix** [livewire/livewire#2345] [1. Help]
Is it possible to create a multiple select dropdown, with livewire and tailwind? I need a multiple select component to assign multiple permissions to a single role. Please help me someone, I need this one. Thanks already!

**Comment by joshhanley** [comment]
@toxifiedmatrix Livewire supports a select with multiple on it. If you need some custom styling you will need to build your own select check out this video from Dan Harrin https://www.youtube.com/watch?v=L_f9gNaSCRE

Hope this helps!
